packages/makeup-hoist-element/src/index.js

```diff
@@ -0,0 +1,70 @@
+let hoistEl;
+let hoistedElementPlaceholder;
+let containerDiv;
+let bodyChildIndexes = [];
+const BODY_TAG_NAME = 'BODY';
+
+function isHoisted(element) {
+    if (element.parentElement.tagName === BODY_TAG_NAME) {
+        return true;
+    }
+    return false;
+}
+
+function unhoist() {
+    if (hoistEl) {
+        if (containerDiv) {
+            [...containerDiv.childNodes].forEach((child) => {
+                if (!child.src) {
```

Comment:
Possibly no need to check again if these elements already got filtered out during hoist()